roles/roles-utils/rpc/src/mini_rpc_client.rs

```diff
@@ -31,6 +31,16 @@ impl MiniRpcClient {
         MiniRpcClient { client, url, auth }
     }
 
+    pub async fn health(&self) -> Result<(), RpcError> {
```

Comment:
probably a longer term refactor, but we should probably use: https://github.com/rust-bitcoin/rust-bitcoincore-rpc 
 Or no?